client/client-multi/client-c/src/kaa/platform-impl/esp8266/esp8266_kaa_client.c

```diff
@@ -60,6 +60,10 @@ static kaa_service_t OPERATIONS_SERVICES[] = { KAA_SERVICE_PROFILE
                                              };
 static const int OPERATIONS_SERVICES_COUNT = sizeof(OPERATIONS_SERVICES) / sizeof(kaa_service_t);
 
+/* Logging constraints */
+#define MAX_LOG_COUNT           SIZE_MAX
+#define MAX_LOG_BUCKET_SIZE     (8 * 1024)
```

Comment:
Please, check whether KAA_MAX_BUCKET_SIZE doesn't exceed the size of a OUT transport buffer.
See defaults.h